css/src/components/gradient-card.scss

```diff
@@ -0,0 +1,34 @@
+$gradient-card-border-radius: $border-radius-lg !default;
+$gradient-card-padding: $layout-2 !default;
+$gradient-card-icon-inline-spacing: 0.375em !default;
+$gradient-card-icon-block-spacing: 0.2em !default;
+
+.gradient-card {
+	border: $border-width-md solid transparent;
+	border-radius: $gradient-card-border-radius;
+	background: linear-gradient(90deg, $gradient-vivid-start, $gradient-vivid-end) border-box;
+	overflow: hidden;
+
+	.gradient-card-content {
+		padding: $gradient-card-padding;
+		background-color: $body-background;
+
+		.gradient-card-title {
+			display: flex;
+			align-items: center;
+			justify-content: flex-start;
+			font-weight: $weight-semibold;
+
+			&:has(+ *) {
```

Comment:
I just saw that update, and wondering what you think about applying `margin-block-start` to all `gradient-card-content` direct children but the first one? Similar to how it's done in `card` component.

```scss
$gradient-card-content-spacing: $layout-2 !default;

.gradient-card-content {
	padding: $gradient-card-padding;
	background-color: $body-background;

	...

	> *:not(:first-child) {
		margin-block-start: $gradient-card-content-spacing;
	}
}
```

That way we fix that:

![image](https://github.com/microsoft/atlas-design/assets/57374379/056e8ab0-621c-42ed-945e-bd0b925687c8)

with proposed changes:

![image](https://github.com/microsoft/atlas-design/assets/57374379/bedb4fc1-1c80-4d05-9df7-65625d3ed984)

Reply:
Seems to me like we should remove gradient card title

Reply:
Removed.